lib/components/Atom/WheelPicker/index.tsx

```diff
@@ -0,0 +1,231 @@
+import {
+    HTMLAttributes,
+    KeyboardEvent,
+    useEffect,
+    useMemo,
+    useRef,
+    useState,
+} from "react";
+import { DropdownItem } from "../dropdown";
+import clsx from "clsx";
+import "./wheel-picker.scss";
+import { InputProps } from "@components/Input/base";
+
+export interface WheelPickerProps extends HTMLAttributes<HTMLElement> {
+    data: { value: string | number }[];
+    selectedValue: string | number;
+    setSelectedValue: (value: string | number) => void;
+    handleKeyDown: (e: React.KeyboardEvent) => void;
+    focus: boolean;
```

Comment:
I'm not the right person, who can give advise regarding naming, but maybe for boolean it'll be better to use `isFocused`? Or something that sound more boolean-like?

Reply:
yes made the change